Cinema/src/test/java/com/example/cinema/service/ProjectionDBServiceTest.java

```diff
@@ -142,11 +142,17 @@ public void UpdateReturningRightData() {
     }
 
     @Test
-    public void UpdateReturnNoIdException(){
```

Comment:
To make difference between Null and Zero. This test is for NoIdException and it throws in two ifs